frontend/src/metabase/dashboard/components/add-card-sidebar/QuestionPickerItem.jsx

```diff
@@ -13,7 +13,7 @@ QuestionPickerItem.propTypes = {
 
 export function QuestionPickerItem({ id, name, icon, isCollection, onSelect }) {
   return (
-    <ItemRoot onClick={() => onSelect(id)}>
+    <ItemRoot role="menuitem" tabIndex={0} onClick={() => onSelect(id)}>
```

Comment:
oops I forgot to mention one final thing to add here -- an `onKeyDown` listener

```
onKeyDown={e => e.key === "Enter" && onSelect(id)}
```

should do the trick
```

Reply:
Sorry for missing that! Added it